enterprise/rules.py

```diff
@@ -30,7 +26,7 @@ def has_implicit_access_to_dashboard(user, obj):  # pylint: disable=unused-argum
         boolean: whether the request user has access or not
     """
     request = get_request_or_stub()
```

Comment:
looking back at the ticket, i see this line: 
> I don’t think `get_request_or_stub` should be a function in `edx-rbac`.  Most likely the client can just use  `crum.get_current_request()` directly.

probably makes sense to take care of that in this PR too.

Reply:
I have added the same function in `enterprise.utils` and updated the import. I do not think we can use `crum.get_current_request` directly, since `edx_rbac.utils. get_request_or_stub` adds some additional functionality. Can you take another look at the changes.

Reply:
i don't think we should conflate testing needs with this functionality - i would prefer we update our tests appropriately instead of baking it into a function we call for production code. i would encourage you to go down the route of calling `crum.get_current_request()` directly here and changing test logic to get the test request object you need.